server/views/applications/prison-dashboard.njk

```diff
@@ -50,7 +50,14 @@
 
       {{ govukPagination(pagination(pageNumber, totalPages, hrefPrefix)) }}
 
+      <p>Applications which meet the following criteria are not shown in the dashboard:
```

Comment:
Is there another referrer dashboard where we need to add this message too?

Reply:
Currently it's on the 'prison dashboard', but not on the dashboards where the referrer looks at their own applications (in progress / submitted). Does the same logic apply to all `get` requests to `/cas2/applications`?

Reply:
Yes it does - submitted applications, prisons and referrer dashboards.

Reply:
This came up during working group. Suzie was looking at the assessor/admin dashboard at `/assess/applications` where the same logic is applied but there's not an explanation that it hides some content for the various reasons.